gcsfs/tests/test_core.py

```diff
@@ -515,6 +515,19 @@ def test_get_put_file_in_dir(protocol, gcs):
         assert gcs.cat(protocol + TEST_BUCKET + "/temp_dir/accounts.1.json") == data1
 
 
+@pytest.mark.parametrize("protocol", ["", "gs://", "gcs://"])
+def test_get_file_to_current_working_directory(monkeypatch, protocol, gcs):
+    fn = protocol + TEST_BUCKET + "/temp"
+    gcs.pipe(fn, b"hello world")
+
+    with tempdir() as dn:
+        os.makedirs(dn)
+        monkeypatch.chdir(dn)
```

Comment:
I used `tempdir` to be consistent with the other tests, but we could simplify this using `tmp_path` fixture and use `monkeypatch.chdir()` on it.

```suggestion
    monkeypatch.chdir(tmp_path)
```